Detections/PulseConnectSecure/PulseConnectSecureVPN-BruteForce.yaml

```diff
@@ -1,27 +1,27 @@
-id: 34663177-8abf-4db1-b0a4-5683ab273f44
-name: Multiple Failed VPN User Logins
-description: |
-  'Creates an incident in the event of multiple unsuccessful attempts to log into the VPN server, which could indicate a potential brute force attack.'
-severity: Low
-requiredDataConnectors:
-  - connectorId: PulseConnectSecure
-    dataTypes: 
-      - Syslog
-queryFrequency: 1h
-queryPeriod: 1h
-triggerOperator: gt
-triggerThreshold: 0
-tactics:
-  - CredentialAccess 
-relevantTechniques:
-  - T1110
-query: |
-
-  let timeframe = ago(1h);
-  let threshold = 20;
-  PulseConnectSecure
-  | where TimeGenerated >= timeframe
-  | where Messages contains "Login failed"
-  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), count() by User, Source_IP
-  | where count_ > threshold
-  | extend timestamp = StartTime, AccountCustomEntity = User, IPCustomEntity = Source_IP
+id: 34663177-8abf-4db1-b0a4-5683ab273f44
+name: Potential Brute Force Attempts
```

Comment:
This rename makes more sense than the other, as the initial has BF identified.  Your name is a generic name, probably should indicate it came from PulseConnectSecure with a name such as - `PulseConnectSecure - Potential Brute Force Attempts`

Reply:
Thanks Shain. Made the suggested changes.